dynawo/sources/Common/Dictionaries/DYNError_en_GB.dic

```diff
@@ -156,6 +156,7 @@ DynamicLineStatusNotSupported =           connection status (CLOSED_1 or CLOSED_
 MismatchingVariableSizes    =             the number of %1% variables and corresponding size are different in model %2% (%3% != %4%)
 ConverterWrongType          =             wrong converter type for converter %1%
 AttemptToPropagateBeforeMerge        =    discrete variables connectors should be merged before propagating information to model
+ZValueIsNaN                 =             z value is NaN
```

Comment:
Please provide more information (at least the model name and the variable index)

Reply:
This error is thrown in ModelMulti::propagateZModif() and is not associated to any model